### ISSUE TITLE
Option to use alternative emoji

### ISSUE THREAD

**Issue by LanderN** [qklabs/qksms#131]
A lot of people (including myself) don't like the standard android emoji.
Two reasons:
- They don't look like the iPhone versions sometimes, which can lead to confusion
- They are ugly (subjective of course)

We could add the twitter emoji in QKSMS, they are open source, so no licensing issues.
I made a quick implementation (does not yet work in the input fields), but it works reasonably well.

Anybody else interested in this?

**Comment by moezbhatti** [comment]
My only problem with this is that it would increase the app size, even for people who don't care

**Comment by LanderN** [comment]
then we could make it an optional download? In a seperate app, like Textra, or just by downloading it to the internal memory and using it from there?

**Comment by moezbhatti** [comment]
Yeah if we could set it up as an optional download, I'd be 100% okay with that

**Comment by LanderN** [comment]
I changed my quick implementation. Now, the font gets downloaded when the preference is set to true, instead of packaged with the app.
Do you mind if I make a pull request so others can improve upon this?

**Comment by moezbhatti** [comment]
Yeah, make the PR and we'll check it out!

**Comment by M2ck** [comment]
@moezbhatti please add asap Twemoji and Emoji One, both are open sources.
Links: [Twemoji](https://github.com/twitter/twemoji) & [Emojione](https://github.com/Ranks/emojione)

**Comment by seanloosier** [comment]
I'd be willing to help with this, I'm a bit of a newb, I've done some android dev at school. QKSMS is quickly becoming my fav SMS app and IMO the biggest thing it's missing is pretty (of course that's subjective) emoji

**Comment by moezbhatti** [comment]
This might be helpful - https://github.com/vanniktech/Emoji

**Comment by oxr463** [comment]
Any updates on this?

**Comment by Parad0xical** [comment]
Silence and Signal both have iOS-style emoji for Android.  As they're open source, can either of their emoji art be imported?

I prefer QKSMS to Silence so would be very keen to see this aesthetic enhancement, either as an optional download or integrated.

**Comment by oxr463** [comment]
I found that I could use AnySoftKeyboard since it has emojis built-in.
